assets/src/edit-story/app/history/useHistoryReducer.js

```diff
@@ -50,6 +55,7 @@ const reducer = (size) => (state, { type, payload }) => {
       // and clear `offset` and `replayState`.
       return {
         entries: [payload, ...state.entries.slice(state.offset)].slice(0, size),
+        globalHistoryLength: state.globalHistoryLength + 1,
```

Comment:
Should the global history remove one if you undo?

Reply:
I think it should just keep incrementing. Maybe some name would clarify this better. But it's a new version no matter what.

Reply:
I think a user should be able to exit an empty story without hitting this message. What if I create a story, create an element, click undo and try and exit. Global history will be two, as element then undoing that element. This just feels a little weird.

Reply:
Sure, so for now, I will revert the undo/redo logic and only track registers after save/load events. If UX improvement on it was later, we can revert it again.